internal/controller/pkg/revision/runtime_defaults.go

```diff
@@ -65,16 +65,18 @@ func deploymentFromRuntimeConfig(tmpl *v1beta1.DeploymentTemplate) *appsv1.Deplo
 func serviceFromRuntimeConfig(tmpl *v1beta1.ServiceTemplate) *corev1.Service {
 	svc := &corev1.Service{}
 
-	if tmpl == nil || tmpl.Metadata == nil {
+	if tmpl == nil {
 		return svc
 	}
 
-	if tmpl.Metadata.Name != nil {
-		svc.Name = *tmpl.Metadata.Name
-	}
+	if meta := tmpl.Metadata; meta != nil {
```

Comment:
i think this refactoring is still lingering from when we were allowing the entire ServiceTemplate spec also - now that we're back to just the metadata, we can probably put this function back to exactly how it was before. 0 changes 😇